web/styles/settings.css

```diff
@@ -581,7 +606,7 @@ input[type="checkbox"] {
 
     /* make the spinner green like the text and box. */
     .loading_indicator_spinner svg path {
-        fill: hsl(178deg 100% 40%);
+        fill: hsl(0deg 0% 100%);
     }
```

Comment:
This seems like a component that's used in many places. Is it correct to change this for everywhere?  Also, the comment you've made incorrect, as that's not green anymore.